src/status_im2/contexts/chat/photo_selector/style.cljs

```diff
@@ -3,28 +3,29 @@
             [react-native.platform :as platform]))
 
 (defn gradient-container
-  [safe-area]
+  [insets]
   {:width    "100%"
```

Comment:
remove :width "100%"

Reply:
It is not redundant here. Views with position absolute require width to be set.